teraserver/python/services/BureauActif/API/QueryLoginType.py

```diff
@@ -29,11 +39,38 @@ def get(self):
         if current_login_type == LoginType.PARTICIPANT_LOGIN:
             login_infos['login_type'] = 'participant'
             login_infos['login_id'] = current_participant_client.id_participant
+            params = {'participant_id': current_participant_client.id_participant}
```

Comment:
Replace 'participant_id' with 'participant_uuid' and 'current_participant_client.id_participant' with 'current_participant_client.participant_uuid'.